pavelib/assets.py

```diff
@@ -292,6 +301,27 @@ def compile_templated_sass(systems, settings):
         print("\t\tFinished preprocessing {} assets.".format(system))
 
 
+def process_npm_assets():
+    """
+    Process vendor libraries installed via NPM.
```

Comment:
I think you will need a unit test for this method. It does look like there is an existing test_assets file.